doc/user_manual/rom.tex

```diff
@@ -3077,9 +3077,31 @@ \subsubsection{ARMA}
       \item \xmlNode{periods}, \xmlDesc{comma-separated integers, required field}, lists the (fundamental)
         periodic wavelength of the Fourier decomposition for these variables, as in the \xmlNode{Fourier} general node.
     \end{itemize}
+  \item \xmlNode{Multiyear}, \xmlDesc{node, optional}, indicates that each sample of the ARMA should yield
+    multiple sequential samples. For example, if an ARMA model is trained to produce a year's worth of data,
+    enabling \xmlNode{Multiyear} causes it to produce several successive years of data. Multiyear sampling
+    is independent of ROM training, and only changes how samples of the ARMA are created.
+
+    \nb The output of a multiyear ARMA must be stored in a \xmlNode{DataSet}, as the targets will depend
+      on both the \xmlNode{pivotParameter} as well as the year, \xmlString{Year}. The year is a second
+      \xmlNode{Index} that all targets should depend on, with variable name \xmlString{Year}.
+
+    The \xmlNode{Multiyear} node accepts the following subnodes:
+    \begin{itemize}
+      \item \xmlNode{years}, \xmlDesc{integer, required field}, the number of years the ARMA should produce
+        each time it yields a sample.
+      \item \xmlNode{growth}, \xmlDesc{float, optional field}, if provided then the histories produced by
+        the ARMA will be multiplied by the growth factor for successive years; that is, the scaling factor
+        is the value given here raised to the power of the year. For example, with a growth factor of 1.5,
+        the first year would not be scaled, the second year would be multiplied by 1.5, the third year by 2.25,
+        and so forth. This simulates growth in the mean value and proportional signal fluctuations each year.
+    \end{itemize}
```

Comment:
This works fine for me, but I still think the 'growth rate' should be 0.5 or 50% for this case.

Reply:
That makes sense to me; that way a negative input would indicate shrinking, and positive growing, right?

I'll ask some of the users and see if they like percentage or decimal more.

Reply:
The summary was they want to put the decimal value, and as you suggested, so for example `50` means scaling by 1.5, and `-50` means scaling by 0.5.

I also went ahead and extended this to specific targets as well.